toffy/panel_utils.py

```diff
@@ -57,7 +57,7 @@ def convert_panel(panel_path):
 
     # read in metadata contained in the first few lines
     col_index = 0
-    with open(panel_path) as r:
+    with open(panel_path, encoding='utf-8') as r:
```

Comment:
Are there any other places in the repo where this might be a problem? I think most instances are wrapped in the `load_json` function which automatically does it, but worth taking a look to see

Reply:
This looks like the only remaining place.